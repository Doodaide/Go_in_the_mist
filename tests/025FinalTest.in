Never gonna give you up
Never gonna let you down
Never gonna go around and hurt you 

Get stick bugged lol!

place hehe
place haha
place ooh 
