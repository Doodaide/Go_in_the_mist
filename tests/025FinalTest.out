Invalid!
Invalid!
Invalid!
Invalid!
Invalid!
Invalid!
Invalid coordinate
Invalid coordinate
Invalid!
